### PR TITLE
fix: OPTIC-1107: [timebox] visual inconsistency with datamanager on smaller screens

### PR DESCRIPTION
**Pull request by yyassi-heartex**
### PR fulfills these requirements
- [x] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [x] Frontend

### Describe the reason for change
addressing a visual issue that occurs when the screen is sufficiently small enough where control bar extends past the edge of all other elements leading to a weird visual
Screenshot 2024-09-12 at 4 33 15 PM

----

| :rocket: | This description was created by [Ellipsis](https://www.ellipsis.dev) for commit d0dda2de29f1992d266d1d82cf1b03511857db6f  | 
|--------|--------|

fix: OPTIC-1107: Adjust app layout for small screens

### Summary:
Fixes layout issue on small screens by adjusting flexbox properties in `login.css` and `App.scss`.

**Key points**:
- **CSS Changes**:
  - In `login.css`, added `flex: 1;` to `html, body` for full height usage.
  - In `App.scss`, added `display: flex;` to `body` and `flex: 1;` to `:global(.app-wrapper)` for consistent layout.
- **Visual Fix**:
  - Fixes issue where control bar extends past other elements on small screens.

----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)

### CODE REVIEW

**Comment by yyassi-heartex** [comment]
@ellipsis-dev review this

File: label_studio/core/static/css/login.css [hunk 1]
```diff
@@ -7,6 +7,7 @@ html, body {
   font-weight: 400;
```
[the remark this answers is not in the document]
Comment (yyassi-heartex) [reply label_studio/core/static/css/login.css]: flex:1 is applied to child element with class `.login_page_new_ui`. Not to body element

[the remark this answers is not in the document]
Comment (yyassi-heartex) [reply label_studio/core/static/css/login.css]: body element is getting `display: flex;` from the css rule updated in `web/apps/labelstudio/src/app/App.scss`

File: web/apps/labelstudio/src/app/App.scss [hunk 2]
```diff
@@ -9,6 +9,11 @@ body {
...
   margin: 0;
   font-family: Figtree, sans-serif;
+  display: flex;
+}
+
+:global(.app-wrapper) {
+  flex: 1;
 }
 
 .app-wrapper {
```
Comment (yyassi-heartex) [inline web/apps/labelstudio/src/app/App.scss:18]: not sure this is actually applying to anything - anyone against removing this block?

File: web/apps/labelstudio/src/app/App.scss [hunk 3]
```diff
@@ -9,6 +9,11 @@ body {
 
   margin: 0;
   font-family: Figtree, sans-serif;
+  display: flex;
```
Comment (bmartel) [inline web/apps/labelstudio/src/app/App.scss:12]: This possibly breaks the other pages, as flex would not be the default layout for body.

**Comment by yyassi-heartex** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10891441407)
> Successfully merged:  create mode 100644 label_studio/ml_models/migrations/0011_thirdpartymodelversion_model_provider_connection.py

**Comment by yyassi-heartex** [comment]
closing this PR as we're using it as a spike - more work to come in another ticket/PR

**Pull request closed without merging**